src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -44,34 +72,36 @@ TopicDomainHeader.propTypes = {
   handleOnMemberPress: PropTypes.func,
   hideSeeMember: PropTypes.bool,
   isFollow: PropTypes.bool,
-  memberCount: PropTypes.number
+  memberCount: PropTypes.number,
+  isLoading: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good practice to provide default values for non-required PropTypes to prevent potential issues with undefined values.
```javascript
TopicDomainHeader.defaultProps = {
  isLoading: false,
};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->